core/electron/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "iTwin.js ElectronHost and ElectronApp",
   "license": "MIT",
   "engines": {
-    "node": ">=12.22.0 < 14.0 || >=14.17.0 <17.0"
+    "node": ">=12.22.0 < 14.0 || >=14.17.0 <17.0 || >= 18.0 < 19.0"
```

Comment:
Can you make sure this matches `core-backend`?

Reply:
Fixed